tests/helpers/screenshot.helpers.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2016-2023 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { expect, Page, TestInfo } from '@playwright/test';
+import * as fs from 'fs';
+import * as path from 'path';
+
+export async function expectScreenshot({
+  component,
+  page,
+  testInfo,
+}: {
+  component: string;
+  page: Page;
+  testInfo: TestInfo;
+}) {
+  const screenshotFilePathParts = [component, `${testInfo.title}.png`];
+
+  trackScreenshot(path.join(...screenshotFilePathParts));
```

Comment:
This is called for each test but this logic could be moved to story-variants.spec.ts and we write the file once after all tests have executed (and hold the file path data in a local variable).

Reply:
I would have to move it when I need a test file for interactions. I don't want everything to be in one test file. And also, right now I am working enabling parallelization for local development. Holding this data in memory won't work with multiple workers.

Reply:
This has been reworked.